packages/mobile/src/images/Images.ts

```diff
@@ -12,6 +12,7 @@ export const circularProgressBig = require('src/images/circular-progress-big.png
 export const clockIcon = require('src/images/clock-icon.png')
 export const consumerIncentives = require('src/images/consumerIncentives.png')
 export const contactPermission = require('src/images/contact-permission.png')
+export const copyIcon = require('src/images/copy-icon.png')
```

Comment:
Are we using this icon anywhere? I see there's also an svg that's being used?

Reply:
This icon should appear next to the shortLink, inside the button, as in the Figma designs. Is it not showing up on your app?

Reply:
Valora specific icons should be added to the Valora branding.
See https://github.com/celo-org/celo-monorepo/pull/4282

Also if choosing to use png/jpg instead of svg, we should provide the 5 different versions  `@1x`, `@1.5x`, `@2x`, `@3x` and `@4x` to cover both iOS and Android screen densities.

Reply:
Would it be better to use SVG?